Makefile

```diff
@@ -140,6 +141,11 @@ ol: OL_JS = ol.js ol-debug.js
 ol: scripts/ol-geoadmin.json .build-artefacts/ol3 .build-artefacts/ol-requirements-installation.timestamp
 	cd .build-artefacts/ol3; \
 	git reset HEAD --hard; \
+	ifdef ${OL3_VERSION} \
+	git checkout ${OL3_VERSION}; \
+	else \
+	git checkout $(OL3_VERSION); \
+	enif; \
```

Comment:
it actually works the same way, you can still use `make OL3_VERSION=bla ol` good to know. making additional tests